src/camera.h

```diff
@@ -91,6 +92,36 @@ class Camera
 		return m_camera_offset;
 	}
 
+	inline void setFov(f32 fov)
+	{
+		m_cache_fov = fov;
+	}
```

Comment:
Quite nonsense to have getter and setter functions for `m_cache_fov` and `m_cache_zoom_fov`. If there were boundary checks, this would be acceptable but how it is right now you could make them public as `cache_zoom`.